commerce-product-test-util/src/main/java/com/liferay/commerce/product/test/util/CPTestUtil.java

```diff
@@ -188,6 +197,40 @@ public static void buildCPInstances(CPDefinition cpDefinition)
 			cpDefinition.getCPDefinitionId(), serviceContext);
 	}
 
+	public static SearchContext getSearchContext(
+		String keywords, int status, Group group) {
+
+		SearchContext searchContext = new SearchContext();
+
+		LinkedHashMap<String, Object> params = new LinkedHashMap<>();
+
+		if (Validator.isNotNull(keywords)) {
+			params.put("keywords", keywords);
+		}
+		else {
+			params.put("keywords", StringPool.STAR);
+		}
+
+		Map<String, Serializable> attributes = new HashMap<>();
+
+		attributes.put(Field.STATUS, status);
```

Comment:
These lines:

    Map<String, Serializable> attributes = new HashMap<>();

    attributes.put(Field.STATUS, status);

should be above this line

    LinkedHashMap<String, Object> params = new LinkedHashMap<>();

The only reason we need a `LinkedHashMap<String, Object> params` variable is to use it in `attributes.put("params", params)`, so it should be declared as close to there as possible.